commodore/helpers.py

```diff
@@ -73,13 +74,16 @@ def _represent_str(dumper, data):
     return dumper.represent_scalar("tag:yaml.org,2002:str", data, style=style)
 
 
-def yaml_dump(obj, file):
+def yaml_dump(obj, file=None):
     """
     Dump obj as single-document YAML
     """
     yaml.add_representer(str, _represent_str)
-    with open(file, "w") as outf:
-        yaml.dump(obj, outf)
+    if file is None:
+        yaml.dump(obj, sys.stdout)
+    else:
+        with open(file, "w") as outf:
+            yaml.dump(obj, outf)
```

Comment:
This seems unnecessary given the latest version, as you're always passing a value for `file`.

Reply:
The file value is from an optional command line option, which can be undefined/None.